substrate-sgx/externalities/src/vectorize.rs

```diff
@@ -0,0 +1,61 @@
+//! Converts maps to vecs for serialization.
```

Comment:
Nice, and elegant solution. 😄

Reply:
The solution itself might be elegant, but not the fact that I had to copy-paste the crate code into here 🙈